cmake/modules/FindJAEGER.cmake

```diff
@@ -0,0 +1,112 @@
+#.rst:
```

Comment:
could you rename `FindJAGER.cmake` to `FindJaeger.cmake`. i think it'd be better to use the exact name used by upstream. and rename the variables and target accordingly?